src/server/scriptVersionCache.ts

```diff
@@ -684,7 +682,8 @@ namespace ts.server {
 
             // Skipped all children
             const { leaf } = this.lineNumberToInfo(this.lineCount(), 0);
-            return { oneBasedLine: this.lineCount(), zeroBasedColumn: leaf ? leaf.charCount() : 0, lineText: undefined };
+            if (!leaf) return { oneBasedLine: 1, zeroBasedColumn: 0, lineText: undefined };
+            else return { oneBasedLine: this.lineCount(), zeroBasedColumn: leaf.charCount(), lineText: undefined };

```

Comment:
Personally, I don't think the `else` improves readability.

Reply:
Yeah, I debated whether to have it or not.  It's gone now.